docs/deploying.md

```diff
@@ -400,6 +400,12 @@ secrets.
 > authentication to work.
 {:.warning}
 
+> **Warning**
+> The official registry image **only** supports htpasswd credentials in
```

Comment:
does this apply only to the _image_ or to the registry in general?

Somewhat wondering 

- if we need to keep both of these as a "WARNING"; perhaps both should be a regular paragraph, describing what's supported and what not - which could be a table of supported hashing algorithms (there could be an extra line added to the "step 1." below.
- would there be a way for the registry to detect a mis-configured `htpasswd` to prevent https://github.com/distribution/distribution/issues/2817#issuecomment-762736783 ?